adserver/api/views.py

```diff
@@ -74,6 +74,8 @@ class AdDecisionView(GeoIpMixin, APIView):
             The number and order must correspond to the ``div_ids``.
```

Comment:
We should probably remove the mention of `Supports the same parameters as via POST with the following changes:` now I think, and just include the full content here?